lib/Doctrine/Common/Cache/RedisCache.php

```diff
@@ -149,20 +155,4 @@ protected function doGetStats()
             Cache::STATS_MEMORY_AVAILABLE  => false,
         ];
     }
-
-    /**
-     * Returns the serializer constant to use. If Redis is compiled with
-     * igbinary support, that is used. Otherwise the default PHP serializer is
-     * used.
-     *
-     * @return int One of the Redis::SERIALIZER_* constants
-     */
-    protected function getSerializerValue()
-    {
-        if (defined('Redis::SERIALIZER_IGBINARY') && extension_loaded('igbinary')) {
-            return Redis::SERIALIZER_IGBINARY;
```

Comment:
The patch breaks BC for those using `Redis::SERIALIZER_IGBINARY`

Reply:
I think I may have found a solution. Could you have a look?

(I can later on squash all commits into a single commit.)

Reply:
Afaik igbinary is a drop-in replacement for PHP's serialize/unserialize functions. So if you have igbinary installed then serialize/unserialize will automatically use the igbinary format. So the new RedisCache will automatically use igbinary if it's available (just like the old RedisCache). So as far as I understand it there shouldn't be a BC break.

Reply:
That only occurs when using php native serialization infrastructure not the serialize/unserialize functions.

Reply:
I've added a commit that should fix this.